src/components/Selection/selectionContainer.js

```diff
@@ -0,0 +1,30 @@
+import { connect } from 'react-redux'
```

Comment:
I think this file can simply be called `container`, since it's inside the `selection` folder? And same thing for the `selectionReducer`.